### PR TITLE
add business icon (building-2)

### PR DESCRIPTION
**Pull request by maxim-s-barabash**
company_3_1

Screenshot from 2022-02-05 00-57-26

Icon Requests from Feather #119
Screenshot from 2022-03-28 19-13-29

### CODE REVIEW

**Comment by ericfennis** [comment]
Welcome to the lucide community @maxim-s-barabash!
Is this not a duplicate of [`building`](https://lucide.dev/icon/building) ?

**Comment by maxim-s-barabash** [comment]
no.  this icon is associated with companies and businesses

**Comment by maxim-s-barabash** [comment]
@ericfennis, what are the chances that he will be accepted?

**Comment by ericfennis** [comment]
hmm. I think this feels a bit as a second building icon to me.
What do you think @locness3

**Comment by mittalyashu** [comment]
This gives more of an enterprise feeling 😅

**Comment by Bowero** [comment]
Personally I agree that this has different use cases than `building`.

**Comment by maxim-s-barabash** [comment]
I renamed the icon

**Comment by delnyn** [comment]
Welcome! Thanks for your contribution. But the icon violates the guidelines. We can probably make it slightly different to make it stick to the guidelines?

**Comment by delnyn** [comment]
image
here are some ideas

**Comment by ericfennis** [comment]
@delnyn Can you try make a variant with a line on the bottom.

**Comment by delnyn** [comment]
image
Like this?

**Comment by ericfennis** [comment]
Niceee, I think these icons will match better with the existing "building" icons. What do you think @maxim-s-barabash @delnyn ?

**Comment by delnyn** [comment]
totally agree

File: icons/building-2.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M17 9h2c.275 0 .554.004.82.07.268.068.521.2.716.394.195.195.326.448.393.715.067.267.071.546.071.821v11M3 22v-8c0-.275.004-.554.07-.82.068-.268.2-.521.394-.716.195-.195.448-.326.715-.393.267-.067.546-.071.821-.071h2m3 6h4m-4-4h4m-4-4h4m-4-4h4M7 22V4c0-.275.004-.554.07-.82.068-.268.2-.521.394-.716.195-.195.448-.326.715-.393C8.446 2.004 8.725 2 9 2h6c.275 0 .554.004.82.07.268.068.521.2.716.394.195.195.326.448.393.715.067.267.071.546.071.821v18M2 22h20"/>
+</svg>
```
Comment (ericfennis) [inline icons/building-2.svg:13]:
```suggestion
<svg 
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M6 22V4c0-.27 0-.55.07-.82.07-.27.2-.52.4-.72.19-.2.44-.32.7-.39C7.46 2 8.73 2 9 2h7c.27 0 .55 0 .82.07.27.07.52.2.72.4.2.19.32.44.39.7.07.28.07.56.07.83v18H6Z" />
  <path d="M2 14v6c0 1.1.9 2 2 2h2V12H4c-.27 0-.55 0-.82.07-.27.07-.52.2-.72.4-.19.19-.32.44-.39.71A3.4 3.4 0 0 0 2 14Z" />
  <path d="M20.82 9.07A3.4 3.4 0 0 0 20 9h-2v13h2a2 2 0 0 0 2-2v-9c0-.28 0-.55-.07-.82-.07-.27-.2-.52-.4-.72-.19-.19-.44-.32-.71-.39Z" />
  <path d="M10 6h4" />
  <path d="M10 10h4" />
  <path d="M10 14h4" />
  <path d="M10 18h4" />
</svg>
```

**Review by ericfennis — CHANGES_REQUESTED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**